tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -266,7 +270,11 @@
     "SpecificVersions": [
       "3.6.*",
       "3.*.*"
-    ]
+    ],
+    "VersionConditions": [{
+      "MinVersion": "3.29.0",
+      "ExcludeTargetFrameworks": ["netcoreapp2.1", "netcoreapp3.0"]
+    }]
```

Comment:
Exclude 3.29.0+ from cosmosdb tests on <`netcoreapp3.1`
    
Because [.NET Standard is a lie](https://andrewlock.net/stop-lying-about-netstandard-2-support/)